.circleci/config.yml

```diff
@@ -5,15 +5,15 @@ orbs:
 
 executors:
   ubuntu:
-    working_directory: ~/go/mattermost/
+    working_directory: ~/mattermost/
```

Comment:
Why does the workflow have to run under a specific directory? Why not let circleci pick one?

Maybe my question is out of scope of this PR, but I'm wondering if we can simplify thing here.

Reply:
I think we need this to persist data and attach the workspace in the next build job.

Reply:
@cpanato Could you confirm that we need to set `working_directory` every time?